tests/TestCase.php

```diff
@@ -56,13 +56,13 @@ protected function getEnvironmentSetUp($app)
         ]);
     }
 
-    protected function assertFileOnDiskContains($fileName, $needle)
+    protected function assertFileOnDiskContains($fileName, $needle, bool $isRegex = false)
```

Comment:
Do not add `isRegex`. Instead create a new method `assertFileOnDiskPassesRegex`.

Reply:
I have altered the original method to support regex only (and changed the name accordingly). The only caller to this particular method is the `CreateTest` tests, regex is necessary for all the tests since the difference in Sqlite libraries between different machines may or may not include the `IF TABLE EXISTS` before the table name depending on the version.

(Note: the original tests in master never passed on my laptop (Mojave, php71 and php72) with either the most recent or the lowest possible dependencies, that is why I made this change.)